xoinvader/ingame.py

```diff
@@ -184,6 +252,10 @@ def events(self):
 
     def update(self):
         self._collision_manager.update()
+        self.wave.update()
+        if not self.wave.running:
```

Comment:
`EnemyWave` enters stop state after the update, and I think it's for the better to restart it immediately, if after the last update call the wave stopped.